pom.xml

```diff
@@ -232,6 +232,10 @@
                             <name>jboss.remoting.debug-buffer-leaks</name>
                             <value>${leak.debug}</value>
                         </property>
+                        <systemPropertyVariables>
+                         <user.language>en</user.language>
```

Comment:
Romain, we use 4 spaces indentation. I must say, its entertaining that you fixed "formatting" above but in the lines you add you break it :-)